src/components/calcite-shell-center-row/calcite-shell-center-row.tsx

```diff
@@ -0,0 +1,73 @@
+import { Component, Event, EventEmitter, Host, Prop, Watch, h, VNode } from "@stencil/core";
+import { CSS } from "./resources";
+import { CalcitePosition, CalciteScale } from "../interfaces";
+
+/**
+ * @slot action-bar - A slot for adding a `calcite-action-bar` to the panel.
+ * @slot - A slot for adding content to the shell panel.
+ */
+@Component({
+  tag: "calcite-shell-center-row",
+  styleUrl: "calcite-shell-center-row.scss",
+  shadow: true
+})
+export class CalciteShellCenterRow {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * Hide the content panel.
+   */
+  @Prop({ reflect: true }) collapsed = false;
```

Comment:
Do we need this one?

Reply:
I assumed we'd want to be somewhat consistent with shell-panel. True or no?

Reply:
We have collapsed because we have the `action-bar` If we `collapse` this panel what is there? @asangma